libraries/chain/include/graphene/chain/custom_authority_object.hpp

```diff
@@ -59,13 +61,7 @@ namespace graphene { namespace chain {
       custom_authority_object,
       indexed_by<
          ordered_unique< tag<by_id>, member< object, object_id_type, &object::id > >,
-         ordered_unique< tag<by_account_custom>,
-            composite_key<
-               custom_authority_object,
-               member<custom_authority_object, account_id_type, &custom_authority_object::account>,
-               member<custom_authority_object, uint32_t, &custom_authority_object::custom_id>
-            >
-         >
+         ordered_non_unique< tag<by_account>, member< custom_authority_object, account_id_type, &custom_authority_object::account > >
```

Comment:
I need to think this thing thru. Non-unique is needed here to make the code work properly. 
I'm not sure we can replace it somehow.

Reply:
I've wrapped long line. But I still have questions regarding ordered_non_unique index. 

I used this index for fetching custom_authorities of the account. So it is non unique because multiple authorities can be assigned to one account.

If I switch `ordered_non_unique( account_id )` with `ordered_unique( account_id, custom_authority_id )`, then I won't be able to fetch all authorities for specific account_id. 

Also I've found that ordered_non_unique is used in core code: account_balance_object, balance_object, account_transaction_history_object, and couple more. 

Are you sure it will create some issues for us?

Reply:
@abitmore  Could you please check my comment above?

Reply:
> If I switch `ordered_non_unique( account_id )` with `ordered_unique( account_id, custom_authority_id )`, then I won't be able to fetch all authorities for specific account_id.

I think this is not true. You can iterate/traverse the same way regardless.

Reply:
I'll experiment with this one. I'll give you update in couple of hours.

Reply:
done. switched to the ordered_unique.